WordPress/Classes/ViewRelated/NUX/SigninSelfHostedViewController.swift

```diff
@@ -95,12 +94,23 @@ import WordPressShared
     }
 
 
+    /// Displays the specified text in the status label.
+    ///
+    /// - Parameters:
+    ///     - message: The text to display in the label.
+    ///
+    func configureStatusLabel(message: String) {
+        statusLabel.text = message
+    }
+
+
     /// Configures the appearance and state of the forgot password button.
     ///
     func configureForgotPasswordButton() {
         var status = ""
-        if statusLabel.text != nil {
-            status = statusLabel.text!
+
+        if let text = statusLabel.text {
+            status = text
```

Comment:
> I think the "forgot password" button being enabled should follow the same logic than the submit button being enabled.

Not quite.  The forgot password button should only the site url.  
I do agree that checking the text isn't the best approach. I'll sort it out.  

> I'm starting to miss ReactiveCocoa a tiny bit

Lies. ;)